### PR TITLE
Adding Metadata user controls and new Metadata form

### PR DESCRIPTION
**Pull request by Muthenya**

### CODE REVIEW

File: ClimsoftVer4/ClimsoftVer4/ucrMetadataPaperArchive.vb [hunk 1]
```diff
@@ -4,6 +4,11 @@
             'SetUpTableEntry 
             SetUpTableEntry("paperarchive")
 
+            ucrTextBoxFormId.SetTableNameAndField("obselement", "elementId")
+            ucrTextBoxFormId.PopulateControl()
+            ucrTextBoxFormId.SetValue("elementId")
```
Comment (Muthenya) [inline ClimsoftVer4/ClimsoftVer4/ucrMetadataPaperArchive.vb]: @Patowhiz I used **setvalue** on forms where the key control was a textbox and not a combo box. Will this do?

Comment (Patowhiz) [reply ClimsoftVer4/ClimsoftVer4/ucrMetadataPaperArchive.vb]: What was the intention of this. I can see you are passing elementid to setvalue.

**Comment by Patowhiz** [comment]
@opencdms/climsoft-dev-branch  @opencdms/climsoft-developers  you can now get the changes that affect metadata forms from this PR. The metadata should now be able to read and write from the database. Check on the `ucrMetadataInstrument ` sample.

File: ClimsoftVer4/ClimsoftVer4/ucrMetadataPaperArchive.vb (hunk 1 above)
Comment (Muthenya) [reply ClimsoftVer4/ClimsoftVer4/ucrMetadataPaperArchive.vb]: I wasn't sure on the text box "equivalent" for the **SetDisplayAndValueMember**.

**Comment by Patowhiz** [comment]
Are you using it as a combobox? Cause only a combobox has such methods as
of now.

**Comment by Muthenya** [comment]
> Are you using it as a combobox? Cause only a combobox has such methods as of now.

No, I'm using a text box.

File: ClimsoftVer4/ClimsoftVer4/ucrMetadataPhysicalFeature.Designer.vb [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+
+    'NOTE: The following procedure is required by the Windows Form Designer
+    'It can be modified using the Windows Form Designer.  
+    'Do not modify it using the code editor.
+    <System.Diagnostics.DebuggerStepThrough()> _
+    Private Sub InitializeComponent()
+        Me.btnOpenFile = New System.Windows.Forms.Button()
+        Me.lblFeatureImageFile = New System.Windows.Forms.Label()
+        Me.lblFeaturePicture = New System.Windows.Forms.Label()
+        Me.txtFeaturePicture = New System.Windows.Forms.PictureBox()
```
Comment (Patowhiz) [inline ClimsoftVer4/ClimsoftVer4/ucrMetadataPhysicalFeature.Designer.vb]: @Muthenya this is a picture box. The pretext is currently txt, could you please change it after merging my PR

Comment (Muthenya) [reply ClimsoftVer4/ClimsoftVer4/ucrMetadataPhysicalFeature.Designer.vb]: What should I change it to?

Comment (Muthenya) [reply ClimsoftVer4/ClimsoftVer4/ucrMetadataPhysicalFeature.Designer.vb]: I have the prefix now as "pb".

**Comment by Patowhiz** [comment]
@Ogik99 @Ivanluv @Muthenya @Wycklife I have made a PR to Lazo Dev. It affects the metadata forms that you were working on. Please go through the changes and compare with what you previously had.

**Pull request merged**